bzl/setup_deps.bzl

```diff
@@ -0,0 +1,33 @@
+load("@bazel_skylib//:workspace.bzl", "bazel_skylib_workspace")
+load("@io_bazel_rules_scala//scala:toolchains.bzl", "scala_register_toolchains")
+load("@io_bazel_rules_scala//scala:scala.bzl", "scala_repositories")
+load("@rules_jvm_external//:defs.bzl", "maven_install")
+load("@build_bazel_rules_apple//apple:repositories.bzl", "apple_rules_dependencies",)
+load("@build_bazel_rules_swift//swift:repositories.bzl", "swift_rules_dependencies",)
+load("@build_bazel_apple_support//lib:repositories.bzl", "apple_support_dependencies",)
+load("@io_bazel_rules_kotlin//kotlin:kotlin.bzl", "kotlin_repositories", "kt_register_toolchains")
+
+def djinni_setup_deps():
+
+    bazel_skylib_workspace()
+
+    scala_register_toolchains()
+    scala_repositories()
+
+    maven_install(
```

Comment:
this really should probably have an explicit name in case a top level project already uses the default `mvn`...

probably not for this patch though @LiFengSC